README.md

```diff
@@ -229,6 +229,30 @@ end
 
 Note that InSpec AWS assumes full compatibility with the underlying AWS SDK and unsupported operations will cause failures.  Therefore, depending on the external provider implementation your mileage may vary!
 
+### `aws_retry_limit` and `aws_retry_backoff`
+
+In certain cases AWS may implement rate limiting. In order to mitigate this issue the `Retry Limit` and `Retry Backoff` can be set in two ways:
+#### 1) Environment Variables
+Setting `AWS_RETRY_LIMIT` and `AWS_RETRY_BACKOFF` environment variables will be implemented at session level
+
+```bash
+export AWS_RETRY_LIMIT=5
+export AWS_RETRY_BACKOFF=5
+```
+#### 2) Inspec Control
+Certain Inspec AWS resources may support setting the Retry Limit and Retry Backoff as shown below.
```

Comment:
Which resources don't support this? I thought all resources would be fine with this, since it's added to the base classes constructor.

Reply:
You're correct, this is actually supported by all resources. This was a misunderstanding on my part.